ivn/core.py

```diff
@@ -337,6 +338,15 @@ def create_interface_d(self):
         self.logger_topo.info("namespace {} interfaces are defined in {}.".
                               format(self.name, ns_network_dir))
 
+    def create_routes(self):
+        for route in self.__ns_info["routes"]:
```

Comment:
```suggestion
        for route in self.__ns_info.get("routes", []):
```
so that it is compatible to previous definition or using default route.

Reply:
we've modified it.